.github/workflows/sql-jdbc-test-and-build-workflow.yml

```diff
@@ -15,24 +15,24 @@ jobs:
           - 17
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
+      - uses: actions/checkout@v3
```

Comment:
`git` says that there are only whitespace changes. Do we need them?

Reply:
This was unintended. Will revert.